browser/app/js/components/Browse.js

```diff
@@ -497,338 +508,349 @@ export default class Browse extends React.Component {
     }
 
     if (web.LoggedIn()) {
-      if (!(used === 0 && free === 0)) {
-        storageUsageDetails = <div className="feh-usage">
-          <div className="fehu-chart">
-            <div style={ { width: usedPercent } }></div>
-          </div>
-          <ul>
-            <li>
-              <span>Used: </span>
-              { humanize.filesize(total - free) }
-            </li>
-            <li className="pull-right">
-              <span>Free: </span>
-              { humanize.filesize(total - used) }
-            </li>
-          </ul>
-        </div>
-      }
-
-
+      storageUsageDetails = <div className="browser-status">
+                              <div className="browser-status__storage">
+                                <small>{ humanize.filesize(total - free) } of { humanize.filesize(total) } Used</small>
+                                <div className="browser-status__chart">
+                                  <div style={ { width: usedPercent } }></div>
+                                </div>
+                              </div>
+                            </div>
     }
 
     let createButton = ''
     if (web.LoggedIn()) {
-      createButton = <Dropdown dropup className="feb-actions" id="fe-action-toggle">
-                       <Dropdown.Toggle noCaret className="feba-toggle">
-                         <span><i className="fa fa-plus"></i></span>
+      createButton = <Dropdown dropup className="create-new" id="dropdown-create-new">
+                       <Dropdown.Toggle noCaret className="create-new__toggle">
```

Comment:
It's fine for me, running MacOS 10.12.5, Firefox 54.0.1(64bit). Also checked in latest Chrome and Safari, in all it's rendered the same. Could you please re-validate?

![icon](https://user-images.githubusercontent.com/13393018/28633210-b1c8a106-7250-11e7-8b62-12ace92aba5a.png)